tests/runcards/protocols.yml

```
@@ -144,6 +144,31 @@ actions:
       nshots: 10
 
 
+  - id: resonator flux dependence flux
```

Just to make it clearer

```suggestion
  - id: resonator flux dependence flux single value
```